tests/instancetype/instancetype.go

```diff
@@ -1706,9 +1720,9 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 							Spec: virtv1.VirtualMachineInstanceSpec{
 								Domain: virtv1.DomainSpec{
 									CPU: &virtv1.CPU{
-										Cores:   uint32(2),
+										Cores:   uint32(numCPU),
```

Comment:
Can you create new constants for these as total vCPU count is sockets * cores * threads.

Something like `providedCores` and `providedSockets`?